opba-protocols/xs2a-protocol-tests/xs2a-bdd-sandbox/src/test/java/de/adorsys/opba/protocol/xs2a/tests/e2e/sandbox/SandboxE2EProtocolAisTest.java

```diff
@@ -216,7 +216,7 @@ private String embeddedListMaxMustermanAccounts(Approach expectedApproach) {
             .user_registered_in_opba_with_credentials(OPBA_LOGIN, OPBA_PASSWORD);
 
         when()
-            .fintech_calls_list_accounts_for_max_musterman()
+            .fintech_calls_list_accounts_for_max_musterman_forBank()
```

Comment:
Don't see the purpose of `_forBank` prefix, moreover, it should be snake_case - `_for_bank` but I would prefer to omit it (bank id is too generic - all calls are with bank ID)

Reply:
because fintech_calls_list_accounts_for_max_musterman_for_bank and fintech_calls_list_accounts_for_max_musterman_with_expected_balances result in 
fintech_calls_list_accounts_for_max_musterman_for_bank_with_expected_balances